Eastern-Research-Group/mywaterway
# Feature/11 native monitoring pages

## Related Issues:
* [HMW-11](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-11)

## Main Changes:
* Created a new page for individual monitoring locations.
* Modified `ReactTable` to take a couple of optional parameters and to resize with the window.
* Modified the `DateSlider` component to prevent unnecessary renders.
* Changed the 'More Information' link in the _Past Water Conditions_ accordion items and popups to 'Monitoring Report page'.

## Steps To Test:
1. Go to an individual station's page, e.g. [http://localhost:3000/station/STORET/DOEE/DOEE-ANA01/](http://localhost:3000/station/STORET/DOEE/DOEE-ANA01/) or by following any 'Monitoring Report page' link for a location
2. Test the download feature and the chart function in particular, and also test for proper error handling
3. Ensure the updates to the `DateSlider` and `ReactTable` components cooperate with the rest of the application

## TODO:
- [ ] Refactor some of the more complex JSX into separate components
